src/configs/directory.rs

```diff
@@ -23,6 +23,7 @@ pub struct DirectoryConfig<'a> {
     pub disabled: bool,
     pub read_only: &'a str,
     pub read_only_style: &'a str,
+    pub repo_truncation_symbol: &'a str,
```

Comment:
To avoid breaking existing configurations, I would prefer defaulting to `truncation_symbol` if `repo_truncation_symbol` was not set

```suggestion
    pub repo_truncation_symbol: Option<&'a str>,
```